src/sentry/event_manager.py

```diff
@@ -482,7 +482,10 @@ def save(
             secondary_grouping_config = project.get_option("sentry:secondary_grouping_config")
             secondary_grouping_expiry = project.get_option("sentry:secondary_grouping_expiry")
             if secondary_grouping_config and (secondary_grouping_expiry or 0) >= time.time():
-                with metrics.timer("event_manager.secondary_grouping"):
+                with sentry_sdk.start_span(
+                    op="event_manager",
```

Comment:
Adding `name` to both blocks to make the perf UI better. Notice "unnamed span" in screenshot.
<img width="594" alt="image" src="https://github.com/getsentry/sentry/assets/44410/4881db07-1ce4-4915-b0fd-024c89dd1c5b">

Reply:
tbh, this is one of the most annoying things about tracing/performance

Reply:
Agreed